src/components/explore/forecast/ForecastTab.js

```diff
@@ -0,0 +1,111 @@
+import { useState, useEffect } from "react";
+import i18n from "@dhis2/d2-i18n";
+import PropTypes from "prop-types";
+import DataLoader from "../../shared/DataLoader";
+import DayForecast from "./DayForecast.js";
+import useAppSettings from "../../../hooks/useAppSettings";
+import styles from "./styles/ForecastTab.module.css";
+
+const convertTimezone = (date, timeZone) =>
+  new Date(date).toLocaleString("sv-SE", { timeZone }); // "sv-SE" follows ISO format
+
+const browserTimeZone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+
+const ForecastTab = ({ geometry }) => {
+  const [data, setData] = useState();
+  const { settings, loading } = useAppSettings();
+
+  const [lng, lat] = geometry.coordinates;
+
+  useEffect(() => {
+    fetch(
+      `https://api.met.no/weatherapi/locationforecast/2.0/compact?lat=${lat}&lon=${lng}`
+    )
+      .then((response) => response.json())
+      .then(setData);
+  }, [lng, lat]);
+
+  if (!data || loading) {
+    return <DataLoader height={400} />;
+  }
+
+  const timeZone = settings?.timeZone || browserTimeZone || "Etc/UTC";
+
+  const timeseries = data.properties.timeseries.map(({ time, data }) => ({
+    time: convertTimezone(time, timeZone),
+    data,
+  }));
+
+  const dates = timeseries.reduce((acc, { time }) => {
+    const date = time.slice(0, 10);
+    if (!acc.includes(date)) {
+      acc.push(date);
+    }
+    return acc;
+  }, []);
+
+  return (
+    <>
+      <table className={styles.table}>
+        <thead>
+          <tr>
+            <td></td>
+            <td>{i18n.t("Night")}</td>
+            <td>{i18n.t("Morning")}</td>
+            <td>{i18n.t("Afternoon")}</td>
+            <td>{i18n.t("Evening")}</td>
```

Comment:
Could it be an idea to merge this into one or two time periods for a day? I am not sure to what extent sunny/sky affects health conditions, but if there is little impact, it could be an argument to merge it into one?

Reply:
We could yes. Let's try to get some feedback on it before we change it to one symbol. Maybe the whole forecast should be redesigned to focus on what matters, like upcoming heat waves, or "blocks" of heavy rainfall. I will leave it for your research :-)

Reply:
Sounds great!